samples/drivers/led_pwm/src/main.c

```diff
@@ -48,24 +48,23 @@ static void run_led_test(const struct device *led_pwm, uint8_t led)
 
 	/* Turn LED on. */
 	err = led_on(led_pwm, led);
-	LOG_INF("  Turning on");
 	if (err < 0) {
 		LOG_ERR("err=%d", err);
 		return;
 	}
+	LOG_INF("  Turned on");
 	k_sleep(K_MSEC(1000));
 
 	/* Turn LED off. */
 	err = led_off(led_pwm, led);
-	LOG_INF("  Turning off");
 	if (err < 0) {
 		LOG_ERR("err=%d", err);
 		return;
 	}
+	LOG_INF("  Turned off");
 	k_sleep(K_MSEC(1000));
 
 	/* Increase LED brightness gradually up to the maximum level. */
-	LOG_INF("  Increasing brightness gradually");
```

Comment:
In this case, I think that message was more appropriate than "Increased" printed after the loop, when the app is no longer presenting anything interesting.